recipe/bld.bat

```diff
@@ -1,6 +1,3 @@
 REM Don't build html docs for this since for some weird reason that fails
 "%R%" CMD INSTALL --build --no-html --data-compress=none .
 IF %ERRORLEVEL% NEQ 0 exit 1
-
-"%R%" -e "IRkernel::installspec(sys_prefix = TRUE)"
```

Comment:
Please let me know if I should restore the behaviour of the previous build script here. Chose to omit it for this PR since it was installing into the wrong path anyway.

Reply:
That's fine. Also, note that this is a `noarch` build so this script was only run on `staged-recipes` and is ignored for feedstock builds.